src/client/datascience/jupyter/debuggerVariables.ts

```diff
@@ -191,7 +192,7 @@ export class DebuggerVariables extends DebugLocationTracker
         // Run the get dataframe rows script
         if (!this.debugService.activeDebugSession || targetVariable.columns === undefined) {
             // No active server just return no rows
-            return {};
+            return { data: [] };
```

Comment:
No its cleaner, we don't need to test the types in the code, as ts compiler will tell us exactly what we'll get.